test/xds/xds_telemetry_labels_test.go

```diff
@@ -132,7 +135,9 @@ func (fsh *fakeStatsHandler) HandleRPC(ctx context.Context, rs stats.RPCStats) {
 		if label, ok := fsh.labels.TelemetryLabels[serviceNamespaceKeyCSM]; !ok || label != serviceNamespaceValue {
 			fsh.t.Fatalf("for telemetry label %v, want: %v, got: %v", serviceNamespaceKeyCSM, serviceNamespaceValue, label)
 		}
-
+		if label, ok := fsh.labels.TelemetryLabels[localityKey]; !ok || label != localityValue {
```

Comment:
Now that there's 3 of these...can we build up our expectation and `cmp.Diff`?

Reply:
Ah, good point. Done.